scio-extra/src/main/scala/com/spotify/scio/extra/sparkey/package.scala

```diff
@@ -145,6 +171,24 @@ package object sparkey {
       sparkeySideInput(basePath).map(reader => new CachedStringSparkeyReader(reader, cache))
   }
 
+  private val DEFAULT_NUM_SHARDS: Short = 1
```

Comment:
🎨 we are starting to adopt `DefaultNumShards` for constants instead of the java'ish way